keyboards/otaku_split/rev1/keymaps/default/keymap.c

```diff
@@ -0,0 +1,72 @@
+/* Copyright 2019 takashiski
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+//not work QMK_KEYBOARD_H. why?
+//#include QMK_KEYBOARD_H
+#include "otaku_split.h"
+// Defines the keycodes used by our macros in process_record_user
+enum custom_keycodes {
+  QMKBEST = SAFE_RANGE,
+  QMKURL
+};
+
+enum layers{
+	BASE=0
```

Comment:
You don't actually need to set this to 0.  It defaults to that, and automatically increments up.

```suggestion
	BASE
```